packages/data-format/src/format/rnf_expense/README.md

```diff
@@ -0,0 +1,60 @@
+# Expense format
+
+| Name            | Type      | Need      | Comment                                |
+| --------------- | --------- | --------- | -------------------------------------- |
+| meta            | object    | Mandatory | see below "meta"                       |
+| creationDate    | date-time | Mandatory | Creation date of the request           |
+| periodStartDate | date-time | Mandatory | Starting period of the current expense |
+| periodEndDate   | date-time | Mandatory | End period of the current expense      |
```

Comment:
Here I'm not sure, are both columns really mandatory, in the RN protocol? Maybe not `periodStartDate` ? I'm not sure.